business/istio_validations.go

```diff
@@ -21,177 +21,106 @@ type ObjectChecker interface {
 	Check() models.IstioValidations
 }
 
-// GetServiceValidations returns an IstioValidations object with all the checks found when running
+// GetValidations returns an IstioValidations object with all the checks found when running
 // all the enabled checkers.
-func (in *IstioValidationsService) GetServiceValidations(namespace, service string) (models.IstioValidations, error) {
+func (in *IstioValidationsService) GetValidations(namespace, service string) (models.IstioValidations, error) {
```

Comment:
"GetValidations" function name should be updated in the next line "GetGoFunctionMetric" to show the correct metric

Reply:
Another comment, if valid empty value for second parameter "service", then I guess it should be commented too.